backend/src/api/users/users.api.ts

```diff
@@ -61,10 +61,10 @@ const initUsersApi: FastifyPluginAsync<Options> = async (fastify, opts) => {
     ) {
       const { id } = req.params;
 
-      const isDeleted = await userService.delete(Number(id));
+      const isDeleted = await userService.delete(req.user.id, Number(id));
 
       return rep
-        .status(isDeleted ? HttpCode.OK : HttpCode.NOT_FOUND)
+        .status(isDeleted ? HttpCode.NO_CONTENT : HttpCode.NOT_FOUND)
```

Comment:
I've checked and no_content makes an error on a frontend side. So, let's return OK back. Sorry for that

Reply:
Please do as @Liza-Veis adviced)